### PR TITLE
Adding an issue template to streamline answers.

### PR DESCRIPTION
**Pull request by sjaakd**
I rather had a dialogue before the user writes a new issue, but this is the best next...

### CODE REVIEW

**Comment by chris922** [comment]
Good idea! As far as I know templates like this could be stored in a `.github` subfolder. Would this be better as this is more or less a "configuration" just for GitHub?

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- [ ] Is this an issue (and hence not a question)? 
+
+If this is a question about a how to use MapStruct there are several resources available.
```
Comment (chris922) [inline ISSUE_TEMPLATE.md]: without "about a" ? -> "If this is a question how to use MapStruct ..."?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- Our reference- and api [documentation](http://mapstruct.org/documentation/reference-guide/).
```
Comment (chris922) [inline ISSUE_TEMPLATE.md]: I personally always like to write API in uppercase =)

**Comment by sjaakd** [comment]
> Good idea! As far as I know templates like this could be stored in a `.github` subfolder. Would this be better as this is more or less a "configuration" just for GitHub?

Thanks! I saw other templates as well in the root (`CONTRIBUTING.MD`) so I decided to leave it there.

**Pull request merged**